upbc/protoc-gen-upb.cc

```diff
@@ -1170,19 +1170,25 @@ int WriteExtensions(const protobuf::FileDescriptor* file, Output& output) {
 
   if (exts.empty()) return 0;
 
+  // Order by full name for consistent ordering.
+  std::map<std::string, const protobuf::Descriptor*> forward_messages;
+  auto add_forward_decl = [&](const protobuf::Descriptor* d) {
+    forward_messages[d->full_name()] = d;
+  };
```

Comment:
at this point is a one liner, so I would inline it

Reply:
Done.